.github/workflows/build_cmake.yml

```diff
@@ -21,7 +21,7 @@ jobs:
           }
         - {
             name: "Ubuntu Build", artifact: "Linux.tar.xz",
-            os: ubuntu-20.04,
+            os: ubuntu-latest,
             cc: "gcc-9", cxx: "g++-9",
```

Comment:
```
CMake Error at /opt/hostedtoolcache/cmake/3.18.6/x64/cmake-3.18.6-Linux-x86_64/share/cmake-3.18/Modules/CMakeDetermineCCompiler.cmake:49 (message):
  Could not find compiler set in environment variable CC:

  gcc-9.
```
The supported GCC versions for each runner image are here: https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2404-Readme.md